index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/b08094b2a2.js" crossorigin="anonymous"></script>
+    <title>Personal Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav><i class="fas fa-bars"></i></nav>
+    </header>
+    <section>
+        <h1 class="header">Hey there.
+            <br>I’m Ian.
+           <h2 class="text_color">I’m a software developer</h2>
+        </h1>
```

Comment:
It's really not semantic to add a h2 tag within a h1 element. You may allow only the h1 element to hold all the text.